This package dumps lots of garbage in /usr/include; contain it in a subdirectory.

--- a/CMakeLists.txt	2024-11-24 23:56:40.906436392 -0500
+++ b/CMakeLists.txt	2024-11-24 23:56:46.986447548 -0500
@@ -103,7 +103,7 @@
 ELSE()
    SET(INSTALL_CMAKE_CONFIG_DIR share/cmake/medfile-${MED_STR_VERSION})
 ENDIF()
-SET(INSTALL_INCLUDE_DIR include)
+SET(INSTALL_INCLUDE_DIR include/med)
 
 ## Pre-requisites detection
 # Detect HDF5 first to potentially retrieve MPI automatically:
